### PR TITLE
Upgrade wagtail 4

### PR DESCRIPTION
**Pull request by nickmoreton**
This work allows the package to be tested using the existing test against Wagtail v4

I didn't need to alter any code of the python files.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,14 +16,16 @@ repos:
     hooks:
       - id: flake8
         language_version: python3.8
-  - repo: https://github.com/prettier/prettier
-    rev: 2.0.5
+  - repo: https://github.com/pre-commit/mirrors-prettier
```
Comment (nickmoreton) [inline .pre-commit-config.yaml:19]: These changes here are because pre-commit was complaining.

File: wagtail_jotform/tests/settings.py [hunk 2]
```diff
@@ -74,3 +76,6 @@
     "API_URL": "https://wagtail-jotform.com",
     "LIMIT": 50,
 }
+
+if WAGTAIL_VERSION >= (3, 0):
```
Comment (nickmoreton) [inline wagtail_jotform/tests/settings.py:80]: Silences the Warning given while tests are running.

**Review by nickmoreton — COMMENTED** [review]
See comments for explanation of changes.

**Comment by kevinhowbrook** [comment]
Thanks @nickmoreton and @katdom13  
I'll merge this monday and do a re-release in pypi

**Pull request merged**